src/App.tsx

```diff
@@ -15,36 +17,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_BY_ALPH: number = 1;
+const SORT_BY_LENGTH: number = 2;
+
+const sortGoods = (goods: string[], key: number): string[] => {
```

Comment:
[CODE STYLE] The function name 'sortGoods' doesn't start with a verb. It's recommended to name functions starting with a verb to indicate what they do. A better name might be 'getSortedGoods'.